stream_alert/rule_processor/handler.py

```diff
@@ -54,8 +61,8 @@ def __init__(self, context, enable_alert_processor=True):
 
         self.metrics = Metrics('RuleProcessor', self.env['lambda_region'])
         self.enable_alert_processor = enable_alert_processor
-        self._failed_record_count = 0
-        self._alerts = []
+        self._failed_record_count = Value('I', 0)
```

Comment:
I initially had it importing the whole package, but you know how I hate annoyingly long package names, so I switched it :).  I can add the namespace back for clarity